reference/specs/bindings/blobstorage.md

```diff
@@ -28,9 +28,64 @@ spec:
 
 ## Output Binding Supported Operations
 
-* create
+* [create](#create-blob)
+* [get](#get-blob)
 
-## Additional information
+
+## Create Blob
+
+To perform a get blob operation, invoke the Azure Blob Storage binding with a `POST` method and the following JSON body:
+
+```json
+{
+  "operation": "create",
+  "data": {
+    "field1": "value1"
+  }
+}
+```
+
+Example:
+
+```bash
+curl -XPOST http://localhost:<dapr-port>/v1.0/bindings/<binding-name> -d '{ "operation": "create", "data": { "field1": "value1" }}'
```

Comment:
nit, data first for readability. Also `POST` is implicit with `-d`

```shell
curl -d '{ "operation": "create", "data": { "field1": "value1" }}' \
       http://localhost:<dapr-port>/v1.0/bindings/<binding-name>
```

Reply:
Changed.